### PR TITLE
show the solution stats in a separate component

### PR DESCRIPTION
**Pull request by KavithaKommuri**

### CODE REVIEW

File: src/app/components/activity-details/activity-details.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div style="border: 2px solid grey;">
```
Comment (wflemingnz) [inline src/app/components/activity-details/activity-details.component.html:1]: style to css file

File: src/app/services/request-manager.service.ts [hunk 2]
```diff
@@ -95,30 +95,35 @@ export class RequestManagerService {
...
+                true
+            )
         );
     }
 
     getStatus(statusUrl: string) {
         return this.http
             .get<FunctionStatusResponse>(statusUrl)
-            .pipe(map((statusResponse) => statusResponse.runtimeStatus));
+            .pipe(map((statusResponse) => statusResponse));
```
[the remark this answers is not in the document]
Comment (KavithaKommuri) [reply src/app/services/request-manager.service.ts]: done

File: src/app/services/request-manager.service.ts [hunk 3]
```diff
@@ -95,30 +95,35 @@ export class RequestManagerService {
     pollFunctionUntilComplete(statusUrl: string) {
         return timer(0, this.statusPollDelay).pipe(
             switchMap(() => this.getStatus(statusUrl)),
-            takeWhile((status) => status != FunctionStatus.Completed, true)
+            takeWhile(
+                (status) => status.runtimeStatus != FunctionStatus.Completed,
```
[the remark this answers is not in the document]
Comment (KavithaKommuri) [reply src/app/services/request-manager.service.ts]: done

File: src/app/components/activity-details/activity-details.component.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    notLoadsCount: number;
+    scenarioName: string;
+    totalCost = 0;
+
+    assignSolutionStatistics(solution: any) {
+        this.scenarioName = solution.ScenarioName;
+        this.notLoadsCount = solution.NotLoads.length;
+        solution.Routes.reduce((prev, cur) => {
```
[the remark this answers is not in the document]
Comment (KavithaKommuri) [reply src/app/components/activity-details/activity-details.component.ts]: i have changed it like

` this.totalCost = solution.Routes.map((r) => r.Cost).reduce(
            (prev, cur) => prev + cur
        );`

File: src/app/components/activity-details/activity-details.component.html (hunk 1 above)
Comment (KavithaKommuri) [reply src/app/components/activity-details/activity-details.component.html]: done

**Pull request merged**